bzt/modules/console.py

```diff
@@ -137,6 +141,9 @@ def check(self):
         """
         Repaint the screen
         """
+        self.data_started = True
```

Comment:
It's just dirty hack for quick-started console window. It's enough to try proposed console without a lot of changes.